Diagnose-PrinterBlockTimeline.ps1

```
@@ -0,0 +1,193 @@
+
+# Collect timeline rows as objects { Time (datetime), Type, Detail }.
+$rows = New-Object System.Collections.Generic.List[object]
+function Add-Row { param([datetime]$Time,[string]$Type,[string]$Detail)
+    $rows.Add([pscustomobject]@{ Time = $Time; Type = $Type; Detail = $Detail })
+}
+
+# --- BOOTS (Event Log service started = 6005, close proxy for boot) ----------
+try {
+    $boots = Get-WinEvent -FilterHashtable @{ LogName='System'; Id=6005; StartTime=$since } -ErrorAction Stop
+    foreach ($b in $boots) { Add-Row $b.TimeCreated 'BOOT' 'system start (EventLog 6005)' }
+} catch { }
```

PSScriptAnalyzer (1.25.0)

[warning] Empty catch block is used. Please use Write-Error or throw statements in catch blocks.

(PSAvoidUsingEmptyCatchBlock)